content/docs/command-reference/repro.md

```diff
@@ -37,30 +42,21 @@ and <abbr>caches</abbr> the pipeline's <abbr>outputs</abbr> along the way.
 💡 For convenience, a Git hook is available to remind you to `dvc repro` when
 needed after a `git commit`. See `dvc install` for more details.
 
+[Stage](/doc/command-reference/run) outputs are deleted from the
+<abbr>workspace</abbr> before executing the stage commands that produce them.
 `dvc repro` does not run `dvc fetch`, `dvc pull` or `dvc checkout` to get data
 files, intermediate or final results (except if the `--pull` option is used).
 
-By default, this command checks all [pipeline](/doc/command-reference/dag)
```

Comment:
all not relevant?

Reply:
This paragraph is redundant (both with the intro as well as with the new targets text) and slightly wrong (but that will be corrected for #2024). Removing it aims to make the Description shorter so that the whole doc doesn't grow too much as we introduced the targets text.

Reply:
But I can easily stash the change if you prefer and leave it for later, lmk.

Reply:
Yes, let's make PRs focused if possible.

Reply:
I expect some changes here, but it probably should be only the paragraph about targets?

Reply:
OK, I reverted this. I'm afraid that if we just keep adding paragraphs without reviewing the context we'll end up with lots of redundancy and a text that may be unnecessary long.